### PR TITLE
EOP-274: Updating Momo release EOL info

### PR DESCRIPTION
**Pull request by dkoerichbird**

### CODE REVIEW

**Review by juliebin — APPROVED** [review]

**Review by deepakpn — APPROVED** [review]

**Comment by deepakpn** [comment]
@dkoerichbird let's wait for @balupillai 's feedback on the EOL dates for versions before merging.

**Review by balupillai — APPROVED** [review]

**Pull request merged**
